frontend/cypress/e2e/features/tests/estatistica.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Estatísticas
+    As an administrator
+    I want to view various performance metrics
+    So that I can monitor and analyze the system's usage and its profitability.
```

Comment:
I prefer these phases all in english, but, we made using mix english with portuguese